CMakeLists.txt

```diff
@@ -511,13 +511,24 @@ endif()
 #   - place the CMakeLists.txt in the first-level subdirectory, e.g.
 #     Externals/zlib/CMakeLists.txt (that is: NOT in some Src/ subdirectory)
 #
+include_directories("${CMAKE_CURRENT_BINARY_DIR}/Externals")
 if (_M_X86)
   add_subdirectory(Externals/Bochs_disasm)
 endif()
 add_subdirectory(Externals/cpp-optparse)
 add_subdirectory(Externals/glslang)
 add_subdirectory(Externals/imgui)
 
+set(DISABLE_FRONTENDS ON CACHE BOOL "" FORCE)
```

Comment:
You may want to consider making these qualified upstream in mGBA then modifying them here, (like `MGBA_DISASBLE_DEPS`), etc. These variables can now potentially leak into other dependencies. The build is essentially more fragile now.

Reply:
Should have added this during the initial review, but, this can be a lot to ask for a separate project. I mean, going, "lol just change ur build option flag names", isn't realistic in some scenarios, especially if the emulator is already being used as a lib elsewhere, since that'd be a minor forward-breaking change. Because of that, I definitely won't block on this if that's too much of a chaotic change.

Pinging the CMake wiz @Orphis. Is there a nicer way to avoid this that doesn't involve upstream
changes, or is that the only way in this scenario?

Reply:
It depends on the extent of functionality that this option adds.
Possibly the best option would be to just use `add_subdirectory(Externals/mgba EXCLUDE_FROM_ALL)` without any special option. The final solution will have all the targets, but they won't be built unless they're actually used.